src/lib/route-helpers.js

```diff
@@ -14,30 +14,30 @@ const promiseResponder = (promise, Component) => (req, res, next) => promise(req
 
 const dateChecker = (d) => !!d && moment.tz(d, dateFormat, true, 'Europe/London').format();
 
-const home = (query, ErrorReporter) => new Promise((resolve) => {
-  const fromDate = dateChecker(query && query.from);
-  if (ErrorReporter && fromDate === 'Invalid date') {
-    throw new ErrorReporter('Must provide "from" date parameter, and optionally a "to" date');
+const dateError = RequestError => (value, message) => {
```

Comment:
I think you could probably do some refactoring here Aaron, can you spot it from the changes we made yesterday?

Reply:
...so having explored this further with you, I think we can ignore this for now.